package.json

```diff
@@ -8,25 +8,18 @@
     "node": ">=5"
   },
   "scripts": {
-    "//":
```

Comment:
looks like npm autoformatted this file. Can you revert the changes to the `scripts` block?